tools/netlist/include/NetlistVisitor.h

```diff
@@ -605,6 +605,14 @@ class NetlistVisitor : public ast::ASTVisitor<NetlistVisitor, true, false> {
         ast::EdgeKind edgeKind = ast::EdgeKind::None;
         if (symbol.procedureKind == ast::ProceduralBlockKind::AlwaysFF ||
             symbol.procedureKind == ast::ProceduralBlockKind::Always) {
+            // auto& body = symbol.getBody();
+            if (symbol.getBody().kind == ast::StatementKind::Block) {
```

Comment:
Do you want to just ignore all statements that aren't a TimedStatement? Otherwise you'll still crash below.

Reply:
As I'm not sure what elements can be nested in the AST, I would rather get a fault during debugging rather than silently skipping netlist elements by mistake.
At the moment, I'm explicitly checking for those AST elements I know I need to skip, in this case a concurrent assertion under a sequential block.
Can concurrent assertions be found elsewhere?